samsungtvws/art.py

```diff
@@ -111,7 +111,7 @@ def _send_art_request(
         assert self.connection
         event: Optional[str] = None
         sub_event: Optional[str] = None
-        while event != wait_for_event:
+        while event != wait_for_event and event != 'error':
```

Comment:
From what I can gather, I think this change is not nececessary.
```suggestion
        while event != wait_for_event:
```